api/http.go

```diff
@@ -64,14 +63,12 @@ func newHTTPHandler(web3Handler Web3Handler) *hTTPHandler {
 }
 
 func (handler *hTTPHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
-	ctx, span := tracer.NewSpan(req.Context(), "handler.ServeHTTP")
-	defer span.End()
```

Comment:
there are other places calling `tracer.NewSpan()`, should same changes be applied there as well?

Reply:
Yes, can remove this later and use `log.Ctx` instead. But it can be used when there are special needs (such as funcs spent time)

actually `log.Ctx`is an event in the `span`